### PR TITLE
Polish translation

### PR DESCRIPTION
**Pull request by hexmind**
I didn't touch `zone_not_drawable_email_html_message`. I think it makes sense from recipient point of view.

Relates to issue #39.

### CODE REVIEW

**Comment by hexmind** [comment]
updated: : ckchangelog, zone_not_drawable_email_html_message

File: Umweltzone/src/main/res/values-pl/strings.xml [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+    <string name="zone_not_drawable_no_email_app">
+        Brak zainstalowanej aplikacji do obsługi e-mail.
+    </string>
+    <string name="zone_not_drawable_app_chooser_title">Otwórz szkic z &#8230;</string>
+    <string name="zone_not_drawable_email_subject">
+        Aplikacja androidowa Umweltzone - integracja z
+        <xliff:g example="Wiesbaden" id="zoneName">%1$s</xliff:g>
+    </string>
+    <string name="zone_not_drawable_email_html_message">
+        <![CDATA[<p>Dear Sir or Madam.
```
Comment (johnjohndoe) [inline Umweltzone/src/main/res/values-pl/strings.xml]: You missed to translate a small part here.

**Comment by johnjohndoe** [comment]
Happy Hacktoberfest @hexmind. If you have time there is a few strings to translate [here](https://github.com/Umweltzone/roadsigns/issues/7).

**Comment by hexmind** [comment]
https://github.com/Umweltzone/roadsigns/pull/19

**Pull request merged**